drupal/sites/all/libraries/ckeditor/config.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * @license Copyright (c) 2003-2023, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
This is a 3rd-party file and must not be changed

Reply:
reverted back